test/docker/base/run_test_wrapper.sh

```diff
@@ -4,6 +4,8 @@ set -e
 echo "Deleting galaxy user - it may not exist and this is fine."
 deluser galaxy | true
 
+GALAXY_TEST_UID=${GALAXY_TEST_UID:-"1"}
```

Comment:
`: ${GALAXY_TEST_UID:="1"}` is a less repetitive way to do this. (also done below with `$GALAXY_VIRTUAL_ENV`)